public/style.css

```diff
@@ -6,7 +6,7 @@
 
 /* add the img folder with this img 'stars.jpg'  */
 body {
-  background: url('img/stars.jpg') repeat 0 0;
+  background: url(../public/img/stars.jpg) repeat 0 0;
```

Comment:
this url should be wrapped in double quotes `""` https://www.w3schools.com/cssref/css3_pr_background.php